docs/lib/utilities/fragments/hub.md

```diff
@@ -105,19 +105,25 @@ const listener = (data) => {
     switch (data.payload.event) {
     
         case 'signIn':
-            logger.error('user signed in'); //[ERROR] My-Logger - user signed in
+            logger.info('user signed in'); //[ERROR] My-Logger - user signed in
```

Comment:
Can we adjust `//[ERROR]` if we are adjust this to an info?

Reply:
What do you mean?

Reply:
Ohhh, the comment. Yes.